docs-requirements.txt

```diff
@@ -86,9 +86,7 @@ sphinxcontrib-serializinghtml==1.1.5
     # via sphinx
 sphinxcontrib-trio==1.1.2
     # via -r docs-requirements.in
-tomli==2.0.1
-    # via towncrier
```

Comment:
I don't see how this is possible for non 3.11 Python.

Looking at towncrier, it looks like `tomli` will be installed for non 3.11 Python but somehow isn't being locked?

Reply:
Alright, so we simply need to check pip-compile in the linting workflow? Should be simple enough.

Reply:
I'll do this in a followup PR